src/components/professor/questions/modals/AddQuestionModal.vue

```diff
@@ -101,22 +102,25 @@ export default {
   },
 
   methods: {
-    addQuestion () {
+    ...mapActions({
+      newQuestion: 'databases/createQuestion'
+    }),
+
+    createQuestion () {
       if (!this.validateData()) return
 
-      // NOTE Validate question adding.
-      const $addBtn = $(this.$refs.addQuestionButton)
+      const $addBtn = $(this.$refs.createQuestionButton)
       $addBtn.addClass('is-loading')
 
-      axios.post('http://localhost:3000/questions', {
+      const q = {
```

Comment:
`q` is a poor name for a variable, use `questionPayload`, or `questionMeta` or anything more prominent.

Also, bear in mind that some of these properties are going to have their names changed (eg. `question` becomes  `content` as per @zemmao's db blueprint). No need to change it within this PR, but make sure it's in one of the following ones.